spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "ingredient index" do
+  it "list ingredients with there attributes" do
+    apple = Ingredient.create!(name: "apple", cost: 2)
+    bread = Ingredient.create!(name: "bread", cost: 4)
+    clove = Ingredient.create!(name: "clove", cost: 8)
+
+    visit "/ingredients"
```

Comment:
Your testing looks good but can be a little more robust by using `within` blocks to really distinguish where these things appear on the page.